features_module_how_to_share_sites_config_with_oth.md

```diff
@@ -0,0 +1,130 @@
+# Features module: how to share a site’s configuration with others?
+
+If you had a chance to build one or two Drupal sites,  you probably already know that Drupal sites consists of three parts:
+**Code** (modules, themes…) stored in files, **Configuration**  (content types, Views, Panels…) stored in the database, and **Content** (nodes, terms…) stored in the database.
+
+![](images/features/3_parts_drupal.png)
+
+When we collaborate on projects, build a site together, we often want to share our work. As we build Drupal sites, define content types, setup Views and Panels, we end up making a lot of configuration changes. These changes are stored in the DB. Code is easy to share, but there is no easy way to merge changes to the DB. So here is our challenge - how do we share a site’s configuration with others?
+
+One possible solution is to document all the configurations made and repeat them in the new environment. However, this is not a very clever way of working. It takes a lot of time and there are too many opportunities to make mistakes along the way. 
+
+Another possibility is to manually export/import configuration in code, as you can see from the example in View under the options:
+
+![](images/features/export_view.png)
+
+Views, Content types, Panels, and some other modules provide this export/import option.  However, not all of modules gives us this option. In addition, working this way doesn't provide us with the ability to know the dependencies of our configuration. For example, when we export View, we need to export also the content type it depends on, otherwise the View won't work. And when the View is complex, there can be more and more configurations that it relies on. How can we be sure we won't miss anythings?
+
+Our solution and the most efficient option, is to use the Features module!
+
+Features simply packages up all of that configuration into a module, and we can use it like any other Drupal module. In another words: **Features is a module that creates modules (known as Features).**
```

Comment:
Well, from my point of view it is a very helpful sentence :-) I once read it somewhere in a tutorial and I remembered it helped me to understand right away that feature = module. 

@RachelBaram as a new for this "feature stuff" - is it confusing you?